specificatie/BRK-Bevragen/openapi.yaml

```diff
@@ -1599,7 +1613,7 @@ components:
         waarde:
           type: "integer"
           title: "waarde"
-          description: "Oppervlak grootte, in vierkante meters. [1-9][0-9]*|0,[1-4]"
+          description: "Oppervlak grootte, met als eenheid de soortGrootte. [1-9][0-9]*|0,[1-4]"
```

Comment:
Waarom zit een pattern in de description? Het is een integer.

Reply:
Geen flauw idee. Zal ooit meegenereerd zijn. Ik verwijder de pattern uit de description.